docs/concepts/add-in-development-best-practices.md

```diff
@@ -117,18 +120,29 @@ For patterns that you can apply as you develop your first-run experience, see [U
 
 - Minimize data exchanges between the add-in and the Office document. For more information, see [Avoid using the context.sync method in loops](correlated-objects-pattern.md).
 
-## Market your add-in
+## Publish and market your add-in
 
-- Publish your add-in to [AppSource](/partner-center/marketplace-offers/submit-to-appsource-via-partner-center) and [promote it](/partner-center/marketplace-offers/promote-your-office-store-solution) from your website. Create an [effective AppSource listing](/partner-center/marketplace-offers/create-effective-office-store-listings).
+- Create an [effective AppSource listing](/partner-center/marketplace-offers/create-effective-office-store-listings).
 
-- Use succinct and descriptive add-in titles. Include no more than 128 characters.
+  - Use succinct and descriptive add-in titles. Include no more than 128 characters.
 
-- Write short, compelling descriptions of your add-in. Answer the question "What problem does this add-in solve?".
+  - Write short, compelling, and accurate descriptions of your add-in. Answer the question "What problem does this add-in solve?".
 
-- Convey the value proposition of your add-in in your title and description. Don't rely on your brand.
+  - Convey the value proposition of your add-in in your title and description. Don't rely on your brand.
 
 - Create a website to help users find and use your add-in.
 
+- Publish your add-in to [AppSource](/partner-center/marketplace-offers/submit-to-appsource-via-partner-center). Be sure to follow the AppSource [prepublish checklist](/partner-center/marketplace-offers/checklist) and [submission guide](/partner-center/marketplace-offers/add-in-submission-guide).
+
+  - Ensure that your add-in works error-free in all operating systems, browsers, and devices supported by the minimum set of APIs declared in your add-in manifest.
+
+  - Be prepared to provide detailed testing instructions and resources for certification reviewers to verify the functionality of your add-in.
```

Comment:
Are we saying developers should be prepared because it's a two-step process of submission, then be asked for these materials?
```suggestion
  - Provide detailed testing instructions and resources for certification reviewers to verify the functionality of your add-in.
```